File: csharp/MapleStory.OpenAPI/Src/KMS/Dto/Character/Character.cs
using Newtonsoft.Json;
using Base = MapleStory.OpenAPI.Common.DTO;

namespace MapleStory.OpenAPI.KMS.DTO
{
    /// <summary>
    /// 캐릭터 정보
    /// </summary>
    public class CharacterDTO : Base.CharacterDTO
    {
        /// <summary>
        /// 캐릭터 식별자
        /// </summary>
        [JsonProperty("ocid")]

        public override string OCID { get; set; }
    }
}

File: js/src/maplestory/api/tms/dto/character/character.ts
import * as base from '../../../common/dto/character/character';
import { CharacterBody } from '../../response/character/characterBody';

/**
 * 角色辨識器資訊
 */
export class CharacterDto extends base.CharacterDto {
  /**
   * 角色辨識器
   */
  public override ocid: string;

  constructor(obj: CharacterBody) {
    super();

    const { ocid } = obj;

    this.ocid = ocid;
  }
}
